app/code/Magento/Ui/view/base/web/js/grid/data-storage.js

```diff
@@ -267,10 +267,12 @@ define([
          * @param {Object} data - Response data.
```

Comment:
Please add data type hint to method DocBlock like
```suggestion
         * @param {Object | null} data - Response data.
```